File: engine/model/src/main/scala/tools/packet/shop/DestroyHiredMerchantBox.scala
package tools.packet.shop

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class DestroyHiredMerchantBox(private var _ownerId: Int) extends PacketInput {
  def ownerId: Int = _ownerId

  override def opcode(): SendOpcode = SendOpcode.DESTROY_HIRED_MERCHANT
}

File: engine/engine-model/src/main/java/tools/packet/shop/DestroyHiredMerchantBox.java
package tools.packet.shop;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record DestroyHiredMerchantBox(Integer ownerId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.DESTROY_HIRED_MERCHANT;
   }
}